docs/admin/maintenance/ossec_alerts.rst

```diff
@@ -36,6 +29,18 @@ Gmail. The SMTP relay does not have to be on the same domain as the
 destination email address, i.e. smtp.gmail.com can be the SMTP relay and
 the destination address can be securedrop@freedom.press.
 
+What you need:
+
+- The *OSSEC Alert Public Key*
+- The fingerprint of the key used when encrypting OSSEC alerts
```

Comment:
This is from the original language, but feels a bit circuitous -- how about just summarizing these two lines as "The *OSSEC Alert Public Key* and its fingerprint"?